### DISCUSSION TITLE
loxilb is now an emerging project of ebpf.io community

### DISCUSSION THREAD

**Discussion by UltraInstinct14** [loxilb-io/loxilb#55] [Announcements]
loxilb is included as an [emerging project](https://ebpf.io/applications/#loxilb) in the prestigious [ebpf.io](https://ebpf.io).  Thanks to all our contributors and well-wishers for getting us this far. We are just getting started and have a long way to go !!

**Comment by TrekkieCoder** [comment]
Congratulations. Keep up the good work !
